LukeMason33/Rotten-Tomatillos
# Setup initial sass structure

### Is this a fix or a feature?

Feature.

### What is the change/fix?

This PR installs sass version 4.14.1, and converts existing css files into sass structure, and adds sass partials for components that have been implemented.

### Where should the reviewer start?

Review the index.scss file (formerly index.css) in `/src`, and the partials added to `/movie-components`.

### How should this be tested?

After running `npm start` in terminal, ensure styling is being applied correctly.
